flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
The function should probably not be returning "local" at all.  That's more of a detail left for the `FlxBasePreloader.isHostUrlAllowed()` function.  I've got an idea to use an Enum for this, but need to refactor the preloader logic to support it.  Let me put together a PR with what I think makes more sense and we can pick up the discussion there.